### PR TITLE
Pre-process LESS example

### PR DESCRIPTION
**Pull request by gregorypratt**

### CODE REVIEW

File: pre-process-less/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  <script src="https://cdn.jsdelivr.net/g/riot@2.2(riot.min.js+compiler.min.js)"></script>
+
+  <!-- mount this app -->
+  <script>
+    riot.parsers.css.less = function(tagName, css) {
+      var style = '';
+      less.render(css, {}, function(error, output) {
+        style = output.css;
+      });
+      return style;
```
Comment (cognitom) [inline pre-process-less/index.html]: `render` function is sync?

**Comment by gregorypratt** [comment]
If you're asking whether it's synchronous or returns a promise then it's capable of either...

**Comment by cognitom** [comment]
Yeah, thanks. I'd have wanted to know `less.render` is async or sync. If async, the code seems not work. If sync, why `less.render` doesn't return the result simply? Just curious.

I couldn't read the detail in this document...
http://lesscss.org/usage/#programmatic-usage

**Comment by cognitom** [comment]
@gregorypratt We've changed the default branch to `gh-pages` for convenience. Could you pls resend your PR there? Thanks!

**Pull request closed without merging**